docs/changelog.md

```diff
@@ -5,6 +5,343 @@ description: |
+    [page setup guide]($guides/page-setup-guide/#columns))
+  - Added support for automatic [line numbering]($par.line) (often used in
+    academic papers)
+  - Typst's layout engine is now multi-threaded. Typical speedups are 2-3x for
+    larger documents. The multi-threading operates on page break boundaries, so
+    explicit page breaks are necessary for it to kick in.
+  - Paragraph justification was optimized with a new two-pass algorithm.
+    Speedups are larger for shorter paragraphs and range from 1-6x.
+  - Highly reduced PDF file sizes due to better font subsetting (thanks to
+    [@LaurenzV](https://github.com/LaurenzV))
+  - Emoji are now exported properly in PDF
+  - Text show rules now match across multiple text elements
+  - Block-level equations can now break over multiple pages. This behaviour can
+    be disabled via `{show math.equation: set block(breakable: false)}`
+  - Fixed a bug where some fonts would not print correctly on professional
+    printers
+  - Fixed a long-standing bug which could cause headings to be orphaned at the
+    bottom of the page
+
+- Layout
+  - Added support for multi-column floating placement and figures via
+    [`place.scope`] and [`figure.scope`]
+  - Added support for automatic [line numbering]($par.line) (often used in
+    academic papers)
+  - Added [`par.spacing`] property for configuring paragraph spacing. This
+    should now be used instead of `{show par: set block(spacing: ..)}`
+    (**Breaking change**)
+  - Added [`block.sticky`] property which prevents a page break after a block
+  - Added [`place.flush`] function which forces all floating figures to be
+    placed before any further content
+  - Added [`skew`] function
+  - Added `{auto}` option for [`page.header`] and [`page.footer`] which result
+    in an automatic header/footer based on the numbering (which was previously
+    inaccessible after a change)
+  - Added `gap` and `justify` parameters to [`repeat`] function
+  - Added `width` and `height` parameters to the [`measure`] function to define
+    the space in which the content should be measured. Especially useful in
+    combination with [`layout`].
+  - The height of a `block`, `image`, `rect`, `square`, `ellipse`, and `circle`
+    can now be specified in [fractional units]($fraction)
+  - The [`scale`] function now supports non-relative lengths for `x` and `y`.
+    This way an element of unknown size can be scaled to a fixed size.
+  - Fixed a bug which could cause headings to be orphaned at the bottom of the
+    page
+  - Fixed footnotes within breakable blocks appearing on the page where the
+    breakable block ends instead of at the page where the footnote marker is
+  - Fixed empty pages appearing when a [context] expression wraps whole pages
+  - Fixed that `{set block(spacing: x)}` could behave differently than
+    `{set block(above: x, below: x)}`. The values of `block.above` and
+    `block.below` can now be retrieved in context expressions.
```

Comment:
I would rephrase the first sentence and have a separate bullet point for the second:

```suggestion
  - Fixed `{set block(spacing: x)}` behaving differently from
    `{set block(above: x, below: x)}`.
  - The values of `block.above` and `block.below` can now be retrieved
    in context expressions.
```